tasks/release.py

```diff
@@ -718,14 +679,14 @@ def _update_release_json_entry(
     import requests
 
     jmxfetch = requests.get(
-        "https://oss.sonatype.org/service/local/repositories/releases/content/com/datadoghq/jmxfetch/{0}/jmxfetch-{0}-jar-with-dependencies.jar".format(
+        "https://oss.sonatype.org/service/local/repositories/releases/content/com/datadoghq/jmxfetch/{0}/jmxfetch-{0}-jar-with-dependencies.jar".format(  # noqa: FS002
             jmxfetch_version,
         )
```

Comment:
```suggestion
        f"https://oss.sonatype.org/service/local/repositories/releases/content/com/datadoghq/jmxfetch/{jmxfetch_version}/jmxfetch-{jmxfetch_version}-jar-with-dependencies.jar"
```
Unless I missed some reason not to change it.

Reply:
no reason, just that it was a looong line 😄